### PR TITLE
chore: update dependencies, prettier pallet move rpc results, get_module_abi

### PR DESCRIPTION
**Pull request by neutrinoks**
Updated dependencies due to return type change of pallet-move.

How output looks like:
smove_rpc_result_get_module_abi

### CODE REVIEW

**Review by asmie — APPROVED** [review]

File: src/cmd/node/rpc/get_module_abi.rs [hunk 1]
```diff
@@ -3,6 +3,7 @@ use clap::Parser;
 use jsonrpsee::core::client::ClientT;
 use jsonrpsee::http_client::HttpClientBuilder;
 use jsonrpsee::rpc_params;
+use move_vm_backend::abi::ModuleAbi;
```
Comment (Rqnsom) [inline src/cmd/node/rpc/get_module_abi.rs]: Let's move this `ModuleAbi` to `move_vm_backend_common` and then import it from there to keep `smove` free from all the heavy stuff within the `move_vm_backend`

**Review by Rqnsom — APPROVED** [review]
LGTM with a small improvement comment

**Review by Rqnsom — COMMENTED** [review]
Maybe we could append the example of the output in the PR description. It would be nice to see what it looks like now!

File: Cargo.toml [hunk 2]
```diff
@@ -25,5 +25,5 @@ move-package = { git = "https://github.com/eigerco/substrate-move.git" }
 move-vm-runtime = { git = "https://github.com/eigerco/substrate-move.git" }
 move-command-line-common = { git = "https://github.com/eigerco/substrate-move.git" }
 move-binary-format = { git = "https://github.com/eigerco/substrate-move.git" }
+move-vm-backend = { git = "https://github.com/eigerco/substrate-move.git" }
```
Comment (Rqnsom) [inline Cargo.toml]: can be removed

**Review by Rqnsom — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (neutrinoks) [reply Cargo.toml]: Done

**Pull request merged**